packages/dev/core/src/Meshes/transformNode.ts

```diff
@@ -185,7 +185,7 @@ export class TransformNode extends Node {
     public onAfterWorldMatrixUpdateObservable = new Observable<TransformNode>();
 
     constructor(name: string, scene: Nullable<Scene> = null, isPure = true) {
-        super(name, scene);
+        super(name, scene, false);

```

Comment:
@ryantrem should isPure be passed here or not related ?

Reply:
It should not be passed, because the meaning of this flag is basically "this instance is concretely a `TransformNode` (not the `Node` base class, and not a derived class such as `AbstractMesh`). I find this a little confusing and don't love this pattern, but it already exists in the codebase.